client-v2/README.md

```diff
@@ -28,7 +28,7 @@ View [setup.sh](scripts/setup.sh).
 
 ## Developing
 
-To start developing, first get [Janus credentials for `cmsFronts`](https://janus.gutools.co.uk/credentials?permissionId=cmsFronts-dev&tzOffset=0) then run:
+To start developing, first get [Janus credentials](https://janus.gutools.co.uk/multi-credentials?&permissionIds=cmsFronts-dev,capi-api-gateway,frontend-frontend-local-dev&tzOffset=0) then run:
```

Comment:
The timezone UTC offset being included in the URL is going to be a problem in summer. This is a design problem with that Janus endpoint rather than a problem with this link though. It uses the timezone so that the credentials expire at 19:00. At the moment it uses the time zone offset to power this.

Leave it like this, but this has alerted me to the fact that this is an issue with Janus. I have a few ideas for improving this behaviour so keep in mind that we should revisit this link in the future...